rbkit-lib/cpu/cpumapping.h

```diff
@@ -0,0 +1,28 @@
+#include <QString>
+
+#ifndef MAPPING
+#define MAPPING
+
+inline namespace MAPS {
+    enum Map {
+        event_type,
```

Comment:
1. regarding the mapping part, [rveventparser](https://github.com/code-mancers/rbkit-client/blob/master/rbkit-lib/rbeventparser.h#L11) does not have mappings for internal attributes of each frame, like [eventType, className, file...](https://github.com/code-mancers/rbkit-client/blob/master/rbkit-lib/stringutil.h#L10)

2. coding conventions fixed to camelCase